src/commands/test-branch.yml

```diff
@@ -47,6 +47,10 @@ steps:
         SOLIDUS_BRANCH: <<parameters.branch>>
         TEST_RESULTS_PATH: test-results/gems-v3-ruby-v2-7-solidus-<<parameters.branch>>/results.xml
       when: always
+  - store_artifacts:
+      name: 'Solidus <<parameters.branch>>: Store spec screenshots'
+      path: spec/dummy/tmp/screenshots
```

Comment:
since [we use `capybara-screenshot`](https://github.com/solidusio/solidus_dev_support/blob/master/solidus_dev_support.gemspec#L33) the path should be

```
spec/dummy/tmp/capybara
```